x-pack/plugins/enterprise_search/public/applications/shared/setup_guide/cloud/instructions.tsx

```diff
@@ -34,10 +34,16 @@ export const CloudSetupInstructions: React.FC<Props> = ({ productName, cloudDepl
                   values={{
                     editDeploymentLink: cloudDeploymentLink ? (
                       <EuiLink href={cloudDeploymentLink + '/edit'} target="_blank">
-                        edit your deployment
+                        {i18n.translate(
+                          'xpack.enterpriseSearch.setupGuide.cloud.step1.instruction1LinkText',
```

Comment:
Is the 1 in `instruction1LinkText` intentional?

Reply:
Yes, they are all intentional in that they match the parent i18n ID, e.g.:

- `xpack.enterpriseSearch.setupGuide.cloud.step1.instruction1`
- `xpack.enterpriseSearch.setupGuide.cloud.step3.instruction1`
- `xpack.enterpriseSearch.setupGuide.cloud.step5.instruction1`

I numbered them in case we ever update instruction copy to have multiple paragraphs, then we can just do instruction1, instruction2, etc. Not 100% sure if that's correct but it's the way it currently is already